newdle/client/src/components/ParticipantTable.js

```diff
@@ -0,0 +1,120 @@
+import React, {useState} from 'react';
+import {useSelector} from 'react-redux';
+import {Radio, Icon, Label, Message, Popup, Table} from 'semantic-ui-react';
+import {CircularProgressbar} from 'react-circular-progressbar';
+import {serializeDate, toMoment} from '../util/date';
+import {getNewdleDuration, getNumberOfParticipants, getParticipantAvailability} from '../selectors';
+import 'react-circular-progressbar/dist/styles.css';
+import styles from './ParticipantTable.module.scss';
+
+const MAX_PARTICIPANTS_SHOWN = 4;
+
+function formatMeetingTime(startTime, duration) {
+  const endTime = startTime.add(duration, 'm').format('HH:mm');
+  return `${serializeDate(startTime, 'HH:mm')} - ${endTime}`;
+}
+
+function renderParticipantNames(availableParticipants) {
+  if (availableParticipants.length === 0) {
+    return <Message warning>Nobody is available at this time</Message>;
```

Comment:
Maybe this could be a regular div? The Message box doesn't really work here visually